docusaurus/docs/Android/04-compose/04-message-components/06-selected-message-overlay.mdx

```diff
@@ -1,68 +1,72 @@
-# SelectedMessageOverlay
+# SelectedMessageMenu
 
-The `SelectedMessageOverlay` component allows you to show different message options to the user when they select a message in the `MessageList`. This is usually done through long tapping on an item.
+The `SelectedMessageMenu` component allows you to show different message options to the user when they select a message in the `MessageList`. This is usually done by long tapping on a message item.
 
 This is a **stateless component** that you can easily add to your UI if you're building a custom Messages screen. Internally, it sets up the following components:
 
 * Reaction options: Shows a list of reactions the user can use to react to the message. Those reactions are shown as `ReactionOptionItem`s in a `Row`.
-* Selected message: Shows the currently selected message. This UI is much simpler than the message item UI used to show messages in a list.
 * Message options: Shows a list of actions the user can take with the selected message, such as **delete**, **edit**, **reply to** and **start a thread** actions.
 
 Let's see how to use it!
 
 ## Usage
 
-If you're using the [`MessagesScreen`](./01-messages-screen.mdx), the `SelectedMessageOverlay` is already set up for you. To use the `SelectedMessageOverlay` in your custom screens, simply add it to your UI, like so:
+If you're using the [`MessagesScreen`](./01-messages-screen.mdx), the `SelectedMessageMenu` is already set up for you. To use the `SelectedMessageMenu` in your custom screens, simply add it to your UI, like so:
 
 ```kotlin
 // The rest of your UI
 if (selectedMessage != null) {
-    SelectedMessageOverlay(
-        // Message options
-        messageOptions = defaultMessageOptions(selectedMessage, user, listViewModel.isInThread),
-        // Message to show
+    SelectedMessageMenu(
+        // Use a Modifier to customize the appearance
+        modifier = Modifier
+            .align(Alignment.BottomCenter),
+        // Specify the message options you provide to the user
+        messageOptions = defaultMessageOptionsState(selectedMessage = selectedMessage,
+            currentUser = user,
+            isInThread = listViewModel.isInThread),
+        // Specify the message upon which the options are exerted
```

Comment:
Also - use simple and understandable English. `message upon which the options are exterted` is just verbose and doesn't give much to users. If anything, given that most users are probably non-native speakers, this could be confusing.

`// The message you selected` should be enough

Reply:
Chaged to the suggested comment